rr_computer_vision/include/traffic_light.hpp

```diff
@@ -1,25 +1,53 @@
 /**
  * @file rr_traffic_light.hpp
  * @brief Traffic Light Header File
- * @author Jamie Kim
- * @author Jason Leung
- * @author Toni Ogunmade(oluwatoni)
- * @author Adrian Malaran
- * @competition IARRC 2018
+ * @author Yuchi(Allan) Zhao
```

Comment:
You should add my name here as well :) 

Even though you were core author when it came to implementation, best to give credit to anybody who was decently involved in algorithm design & discussion